kanopi/kanopi-pack-vue
# Kanopi Pack v2 Compatability Updates

## Description
> As a developer, I need to update contained packages and ensure interoperability with peer dependencies

- Update supported packages to latest compatible releases
- Change webpack-merge usage to match new pacakage signatures

## Related Tickets

* [Kanopi Pack Dependency Updates](https://kanopi.teamwork.com/app/tasks/29018128)


## Steps to Validate
1. Change the dependency for a project using a previous version to this branch version
2. Test running in Development and Production mode.
3. Ensure asset compilation and any HMR updates appropriate to the mode.
